app/views.py

```diff
@@ -82,13 +96,12 @@ def login():
                                 auth_type='password')
 
         # If regular error, tell user error
+        error_message(token)
         if 'error' in token:
-            flash(token['text'])
             status = token['status']
-        # If pymesync error, tell user vague error
+        # If pymesync error, tell user error
         elif 'pymesync error' in token:
             print token
```

Comment:
Why are we printing here? Actually, is this whole elseif necessary? Seems like error_message() will take care of printing errors, and this block isn't doing anything else.